examples/apollo_server/src/index.ts

```diff
@@ -0,0 +1,102 @@
+// TODO once we have dfx 0.16.x working reenable these tests in CI
+
+import { Server } from 'azle';
+import { ApolloServer } from '@apollo/server';
+import { expressMiddleware } from '@apollo/server/express4';
+import express from 'express';
+
+// A schema is a collection of type definitions (hence "typeDefs")
+// that together define the "shape" of queries that are executed against
+// your data.
+const typeDefs = `#graphql
+  # Comments in GraphQL strings (such as this one) start with the hash (#) symbol.
+
+  # This "Book" type defines the queryable fields for every book in our data source.
+  type Book {
+    id: String
+    title: String
+    rating: Int
+  }
+
+  type Author {
+    id: String
+    name: String
+  }
+
+  # The "Query" type is special: it lists all of the available queries that
+  # clients can execute, along with the return type for each. In this
+  # case, the "books" query returns an array of zero or more Books (defined above).
+  type Query {
+    books: [Book]
+    authors: [Author]
+    titles: [String]
+  }
+
+  type Mutation {
+    createBook(title: String!, rating: Int!): Book
+  }
+`;
+
+let books = [
+    {
+        id: '0',
+        title: 'The Awakening',
+        rating: 34
+    },
+    {
+        id: '1',
+        title: 'City of Glass',
+        rating: 63
+    }
+];
+
+const authors = [
+    {
+        id: '0',
+        name: 'Jordan'
```

Comment:
What if we updated these authors to be the authors of the above books, or updated the titles to be the titles of works created by Jordan or Ben respectively, I don't know how much I shifting the credit to authorship of those books